### PR TITLE
watch

### PR DESCRIPTION
**Pull request by vsupruniuk**
- [DEMO LINK](https://vsupruniuk.github.io/layout_stop-watch/)

### CODE REVIEW

File: src/styles/main.scss [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: $watch-diameter;
+  width: $watch-diameter;
+  background-color: #00f;
+  border-radius: 50%;
+}
+
+.watch__circle {
```
Comment (mflkvsk) [inline src/styles/main.scss:19]: you can use & here (check the same options below)

File: src/styles/main.scss [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+  width: $circle-diameter;
+  background-color: #fff;
+  border-radius: 50%;
+}
+
+.watch__min-arrow {
+  position: absolute;
+  z-index: 2;
+  top: ($watch-diameter / 2) - $minutes-arrow-height;
+  width: 10px;
```
Comment (mflkvsk) [inline src/styles/main.scss:32]: you have a variable  for width

File: src/styles/main.scss [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+$minutes-arrow-width: 10px;
+$minutes-arrow-height: $watch-diameter / 3;
+
+.watch {
+  position: relative;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: $watch-diameter;
+  width: $watch-diameter;
```
Comment (mflkvsk) [inline src/styles/main.scss:14]: you can use mixin for repeatable property

**Review by mflkvsk — APPROVED** [review]

**Review by Olexandr6 — APPROVED** [review]
Good job🔥

**Pull request closed without merging**